Error trying to run in webpack annot resolve module 'utils/Meta'

I get the following error:

````
ERROR in ./~/stardust/dist/collections/Breadcrumb/Breadcrumb.js
Module not found: Error: Cannot resolve module 'utils/Meta' in /path/to/project/node_modules/stardust/dist/collections/Breadcrumb
 @ ./~/stardust/dist/collections/Breadcrumb/Breadcrumb.js 21:12-33
````

It is fixed when I make this change:

```var _Meta = require('utils/Meta');```   -->  ```var _Meta = require('../../utils/Meta');```

Is that a proper fix?

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -131,7 +131,7 @@ Every component has a static property called `_meta`.  This object defines the c
 Here's an example `_meta` object:
 
 ```js
-import META from 'src/utils/Meta'
+import META from '../../utils/Meta'
 
 const _meta = {
   library: META.library.semanticUI,

--- a/src/collections/Breadcrumb/Breadcrumb.js
+++ b/src/collections/Breadcrumb/Breadcrumb.js
@@ -1,7 +1,8 @@
 import _ from 'lodash'
 import cx from 'classnames'
 import React, { PropTypes } from 'react'
-import META from 'src/utils/Meta'
+
+import META from '../../utils/Meta'
 import { customPropTypes, getUnhandledProps } from '../../utils/propUtils'
 import * as sui from '../../utils/semanticUtils'
 import BreadcrumbDivider from './BreadcrumbDivider'

--- a/src/collections/Breadcrumb/BreadcrumbDivider.js
+++ b/src/collections/Breadcrumb/BreadcrumbDivider.js
@@ -1,5 +1,6 @@
 import React, { PropTypes } from 'react'
 import cx from 'classnames'
+
 import META from '../../utils/Meta'
 import { customPropTypes, iconPropRenderer, getUnhandledProps } from '../../utils/propUtils'
 
@@ -17,8 +18,6 @@ function BreadcrumbDivider(props) {
   const rest = getUnhandledProps(BreadcrumbDivider, props)
 
   if (icon) {
-    // TODO: After update <Icon> to API update propName
-
     return iconPropRenderer(icon, { ...rest, className: classes })
   }
 

--- a/src/collections/Breadcrumb/BreadcrumbDivider.js
+++ b/src/collections/Breadcrumb/BreadcrumbDivider.js
@@ -1,5 +1,5 @@
-import React, { PropTypes } from 'react'
 import cx from 'classnames'
+import React, { PropTypes } from 'react'
 
 import META from '../../utils/Meta'
 import { customPropTypes, iconPropRenderer, getUnhandledProps } from '../../utils/propUtils'

--- a/src/collections/Breadcrumb/BreadcrumbSection.js
+++ b/src/collections/Breadcrumb/BreadcrumbSection.js
@@ -1,5 +1,6 @@
-import React, { PropTypes } from 'react'
 import cx from 'classnames'
+import React, { PropTypes } from 'react'
+
 import META from '../../utils/Meta'
 import { customPropTypes, getUnhandledProps, useKeyOnly } from '../../utils/propUtils'
 

